platform-sdk/platform-apps/tests/PlatformTestingTool/src/main/java/com/swirlds/demo/platform/PlatformTestingToolState.java

```diff
@@ -650,7 +666,14 @@ public synchronized ExpectedFCMFamily getStateExpectedMap() {
      */
     @Override
     public synchronized PlatformTestingToolState copy() {
+        //        logger.info(
+        //                LOGM_DEMO_INFO,
+        //                "PlatformTestingToolState.copy() immutable: {}, StackTrace: {}",
+        //                isImmutable(),
+        //                StackTrace.getStackTrace());
+
         throwIfImmutable();
+        //        setImmutable(true);
```

Comment:
I'm not sure if it's safe not to make this state immutable upon copying.

Reply:
Yes, it should be there. Uncommented